mlflow/gateway/client.py

```diff
@@ -344,3 +345,77 @@ def query(self, route: str, data: Dict[str, Any]):
                 raise MlflowException(message=timeout_message, error_code=BAD_REQUEST)
             else:
                 raise e
+
+    @experimental
+    def set_limits(self, route: str, limits: List[Dict[str, Any]]) -> LimitsConfig:
+        """
+        Set limits on an existing route in the Gateway.
+
+        .. warning::
+
+            This API is **only available** when running within Databricks.
+
+        :param route_name: The name of the route to set limits on.
+        :param limits: Limits (Array of dictionary) to set on the route. Each limit is defined by a
+                       dictionary representing the limit details to be associated with the route.
+                       This dictionary should define:
+
+                       - renewal_period: a string representing the length of the window 
+                       to enforce limit on (only supports "minute" for now).
+                       - calls: a non-negative integer representing the number of calls 
+                       allowed per renewal_period (e.g., 10, 0, 55).
+                       - key: an optional string represents per route limit or per user 
+                       limit ("user" for per user limit, "route" for per route limit, if not  supplied, default to per route limit).
+
+        :return: The returned data structure is a serialized representation of the `Limit`
```

Comment:
```suggestion
                       limit ("user" for per user limit, "route" for per route limit, if not 
                       supplied, default to per route limit).
                       
                       
        :return: The returned data structure is a serialized representation of the `Limit`
```
Sphinx is really weird about the space after a list in a docstring. Could we try this?